contribs/gmf/src/datasource/ExternalDataSourcesManager.js

```diff
@@ -471,29 +471,35 @@ const exports = class {
   createAndAddDataSourceFromFile(file, opt_callback) {
     this.getFileDataSource_(file).then(
       (dataSource) => {
+        let success = true;
         const fileGroup = this.fileGroup_;
 
-        // (1) No need to do anything if the file has already been added...
-        if (fileGroup.dataSources.includes(dataSource)) {
-          return;
-        }
+        // Look if the extent is valid (and so at least one geometry)
+        if (dataSource.extent[0] === Infinity) {
+          console.error('The file contains no geometry');
```

Comment:
do we want to keep this `console.error`?

Reply:
Not specially, There is already a warning to the user that says "bad file".
Should I remove it ?